experiments/saxpy/openmp/execute_experiment.tpl

```diff
@@ -4,10 +4,30 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-{batch_nodes}
-{batch_ranks}
-{batch_timeout}
-
 cd {experiment_run_dir}
 
+export N_TASKS="{n_ranks}"
+export N_NODES="{n_nodes}"
+export GPUS_PER_NODE="{gpus_per_node}"
+export CORES_PER_TASK="{cores_per_task}"
```

Comment:
Note: if these are undefined (e.g. not appearing in `variables.yaml` for a given experiment) then the template filling generates the literal `export CORES_PER_TASK="{cores_per_task}"`

`generate-batch` can then infer that the particular experiment does not set that variable.